compiler/modules/front.syntax/tests/testParserOnTestSuite.bal

```diff
@@ -79,7 +79,7 @@ function testParserOnTestSuite() returns err:Syntax|io:Error|file:Error? {
                     test:assertEquals(lines, canonSrc, "serialized ast can't be re-parsed");
                     panic err:impossible("if src is equal to canonSrc second parse can't fail");
                 }
-                string[] roundTripSrc = partToLines(check part);
+                string[] roundTripSrc = partToLines(part);
```

Comment:
This is a good change. But this is not an error according to the spec, so jBallerina should only be giving a warning here.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/33717